### ISSUE TITLE
Can embedded ruby read post data?

### ISSUE THREAD

**Issue by madnik** [iridakos/duckrails#3]
Can the embedded code under advance customization read post data?

**Comment by iridakos** [comment]
Yes, in the advanced tab (and in the Response body tab) the embedded ruby scripts can access both the parameters variable but the original request as well. I updated the guide http://www.arubystory.com/p/duckrails-tutorial.html. Let me know if something is missing.

**Comment by madnik** [comment]
Great it works. For anyone else this is what I was trying to achieve. 

```
<%=
File.open('path_to_file', 'w') { |file| file.write(@parameters[:token]) }
%>
```

You can close this. Bravo!

**Comment by iridakos** [comment]
Wrong number in the commit :P.

Commit c9b859b closes an other issue (#4).
